test/integration/associations/has-many.test.js

```diff
@@ -745,8 +745,10 @@ describe(Support.getTestDialectTeaser('HasMany'), () => {
         await article.addLabel(label1.id);
         await article.setLabels([label2.id]);
         const labels = await article.getLabels();
-        expect(labels).to.have.length(1);
-        expect(labels[0].text).to.equal('label two');
+        expect(labels).to.have.length(2);
```

Comment:
`article.setLabels([label2.id])` means that only label2 will be associated to article, so the new length of labels should be one. The original test is correct